### PR TITLE
Fix some broken links in documentation

### PR DESCRIPTION
**Pull request by errordeveloper**
These came up while testing the website.

### CODE REVIEW

File: site/plugin.md [hunk 1]
```diff
@@ -6,7 +6,6 @@ search_type: Documentation
 
  * [Launching Weave Net and Running Containers Using the Plugin](#launching)
  * [Restarting the Plugin](#restarting)
- * [Configuring the Plugin](#configuring)
```
Comment (errordeveloper) [inline site/plugin.md]: This section has gone away.

File: site/weavedns/managing-entries-weavedns.md [hunk 2]
```diff
@@ -4,17 +4,13 @@ menu_order: 40
 search_type: Documentation
 ---
 
-
 The following topics are discussed: 
 
 * [Adding and removing extra DNS entries](#add-remove)
 * [Resolving WeaveDNS entries from the Host](#resolve-weavedns-entries-from-host)
 * [Hot-swapping Service Containers](#hot-swapping)
-* [Retaining DNS Entries When Containers Stop](#retain-stopped)
```
Comment (errordeveloper) [inline site/weavedns/managing-entries-weavedns.md]: Was this section never written?

**Comment by bboreham** [comment]
Should be on 1.9 branch

**Comment by errordeveloper** [comment]
Can I cherry-pick from master or you want me to make this agains 1.9 and merge into master after it is in 1.9?

**Comment by bboreham** [comment]
The latter, please.

**Comment by abuehrle** [comment]
Here's another broken link in the See Also list on this page.  https://github.com/weaveworks/weave/blob/1.9/site/weavedns/managing-entries-weavedns.md

 * [How Weave Finds Containers](/site/weavedns/how-works-weavedns.md)

File: site/weavedns/managing-entries-weavedns.md (hunk 2 above)
Comment (abuehrle) [inline site/weavedns/managing-entries-weavedns.md]: I looked through the history and I don't see that topic.  I will give it a more thorough look later.

**Comment by bboreham** [comment]
Do you want to fix that extra one ("How Weave Finds Containers") or shall I merge this and you do that separately?

**Comment by bboreham** [comment]
Rebased and fixed that extra link.

**Pull request merged**